CONTRIBUTING.md

```
@@ -54,6 +54,7 @@ This project uses the following technologies:
 - shadcn/ui: https://ui.shadcn.com/
 - v0: https://v0.dev/
 - IndexedDB API Wrapper (dexie.js): https://dexie.org/
+- Cloudflare KV: https://developers.cloudflare.com/api/resources/kv/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)